config/foreman.hiera/tuning/sizes/extra-extra-large.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apache::mod::passenger::passenger_max_pool_size: 240
+apache::mod::passenger::passenger_max_request_queue_size: 1000
+# TODO this is 1/10th of the base
```

Comment:
Good point, do you know how you got 10000 for this value for the base? That seems high.  If you can tell me how you go it, I will work on creating issues to get this fixed.

Reply:
b570b53b8e7df5cb0fdff30029a7e0c1a4c60dff added it. Initially 100000 was suggested in the ticket but per https://projects.theforeman.org/issues/18821#note-2 I asked a trusted user who suggested 10000.